validation/src/main/scala/hmda/validation/rules/IfInstitutionPresentInAggregate.scala

```diff
@@ -0,0 +1,13 @@
+package hmda.validation.rules
+
+import hmda.model.institution.Institution
+import hmda.validation.context.ValidationContext
+
+object IfInstitutionPresentInAggregate {
```

Comment:
This name is misleading, since it's actually checking for both the institution and the year are present. Maybe we could call it `IfContextPresentInAggregate`?